src/DataStructures/Tensor/Expressions/TensorExpression.cpp

```diff
@@ -0,0 +1,25 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "DataStructures/Tensor/Expressions/TensorExpression.hpp"
+
+TensorIndex<0> ti_a;
```

Comment:
In the hpp file there is a note saying that these do not need to be instantiated. What changed? Please update the documentation